doc/00_Linee guida interoperabilità tecnica/progettazione-dei-servizi-digitali/tecnica.rst

```diff
@@ -1,83 +1,69 @@
-|                       | per ogni e-service,   |                       |
-|                       | individui le          |                       |
-|                       | caratteristiche       |                       |
-|                       | funzionali o non      |                       |
-|                       | funzionali per:       |                       |
-+-----------------------+-----------------------+-----------------------+
-|                       | (REG_TECNICA_001.a)   | Selezionare la        |
-|                       |                       | porzione del modello  |
-|                       |                       | dati individuato      |
-|                       |                       | nella fase semantica. |
-+-----------------------+-----------------------+-----------------------+
-|                       | (REG_TECNICA_001.b)   | Individuare le        |
-|                       |                       | tecnologie tra SOAP o |
-|                       |                       | REST.                 |
-+-----------------------+-----------------------+-----------------------+
-|                       | (REG_TECNICA_001.c)   | Individuare i pattern |
-|                       |                       | di interazione.       |
-+-----------------------+-----------------------+-----------------------+
-|                       | (REG_TECNICA_001.d)   | Individuare i pattern |
-|                       |                       | di sicurezza.         |
-+-----------------------+-----------------------+-----------------------+
-| (REG_TECNICA_002)     | Implementare le API.  |                       |
-+-----------------------+-----------------------+-----------------------+
-| (REG_TECNICA_003)     | Esporre ai fruitori   |                       |
-|                       | una funzione utile a  |                       |
-|                       | verificare lo stato   |                       |
-|                       | di funzionamento      |                       |
-|                       | delle API.            |                       |
-+-----------------------+-----------------------+-----------------------+
+.. list-table:: 
+   :widths: 15 40
+   :header-rows: 0
+
+   * - (REG_TECNICA_001)
+     - Dare seguito all’analisi tecnica dei requisiti che, per ogni e-service, individui le caratteristiche funzionali o non funzionali per:
+
+         * (REG_TECNICA_001.a) Selezionare la porzione del modello dati individuato nella fase semantica.
+               
+         * (REG_TECNICA_001.b) Individuare le tecnologie tra SOAP o REST.
+
+         * (REG_TECNICA_001.c) Individuare i pattern di interazione.
+
+         * (REG_TECNICA_001.d) Individuare i pattern di sicurezza.
+
+
+   * - (REG_TECNICA_002) 
+     - Implementare le API.
+
+   * - (REG_TECNICA_003) 
+     - Esporre ai fruitori una funzione utile a verificare lo stato di funzionamento delle API.
+ 
+|
```

Comment:
@vintra73 il pipe alla fine serve o è un refuso?

Reply:
serve per inserire un ritorno a capo, ho verificato che in cado di width basso si abilita una scroll-bar orizzontale che affoga il testo

Reply:
:clap: